Makefile

```diff
@@ -205,6 +212,30 @@ test_nodejs_wasm_api_in_docker: build_nodejs_test_docker
 test_nodejs_wasm_api: build_node_js_api
 	cd tfhe && node --test js_on_wasm_tests
 
+.PHONY: bench_shortint # run benchmarks for shortint
```

Comment:
run -> Run (nitpick but the toolchain selection needs a fix, so I write it here)